Editor/Helper/ButtonMethodAttribute.cs

```diff
@@ -1,163 +0,0 @@
-/// <summary>
-/// ButtonMethodAttribute,
-/// modified from https://github.com/Deadcows/MyBox/blob/master/Attributes/ButtonMethodAttribute.cs
-/// </summary>
```

Comment:
May I ask why this file is breaking down to 3 small files?

Reply:
i think that's more clean to divide logic items and move them to their own location. I can revert that, if u want, np